circuitbreaker/umbrel-app.yml

```diff
@@ -36,7 +36,10 @@ description: >-
 
   WARNING: See queue mode warning.
 releaseNotes: >-
-  - Initial release on the Umbrel App Store
+  - Improved web interface.
+  - Edit limits for multiple nodes in one operation.
+  - Added BLOCKED mode to completely block traffic from a peer.
+  - Fixed bug where Circuit Breaker wouldn't restart after a failure.
```

Comment:
Our umbrel-app.yml has some funny quirks for formatting. If you want each of these to be on a separate line then there needs to be an extra newline between each (see suggestion below). Otherwise it renders like this in the UI:

<img width="852" alt="Screenshot 2023-03-23 at 12 25 32 PM" src="https://user-images.githubusercontent.com/85373263/227329028-3bcd18de-0fca-4b40-92e6-788e61b1a5fc.png">

```suggestion
  - Improved web interface.
  
  - Edit limits for multiple nodes in one operation.
  
  - Added BLOCKED mode to completely block traffic from a peer.
  
  - Fixed bug where Circuit Breaker wouldn't restart after a failure.
```

Reply:
Fixed